Content.Shared/_RMC14/Weapons/Ranged/CMGunSystem.cs

```diff
@@ -636,7 +637,7 @@ private void OnUserBlockShootingInsideContainersAttemptShoot(Entity<UserBlockSho
             args.Cancel();
     }
 
-    private void OnAmmoEjectActivateInWorld(Entity<RMCAmmoEjectComponent> gun, ref ActivateInWorldEvent args)
+    private void OnAmmoEjectActivateInWorld(Entity<RMCAmmoEjectComponent> gun, ref ActivateInWorldEvent args) //
```

Comment:
```suggestion
    private void OnAmmoEjectActivateInWorld(Entity<RMCAmmoEjectComponent> gun, ref ActivateInWorldEvent args)
```
clean up empty comment